### ISSUE TITLE
In Ubuntu 16.04 after building Cura, it doesnt install

### ISSUE THREAD

**Issue by joinalahmed** [Ultimaker/cura-build#69]
@kenjo @Ghostkeeper 
when I open the .deb file wth ubuntu software and try to install it doesn't install.

**Comment by Ghostkeeper** [comment]
Does it give an error when you try to install it?

**Comment by thopiekar** [comment]
Of course it doesn't. There is no Python 3.4 on Ubuntu Xenial anymore.

**Comment by joinalahmed** [comment]
no it just tries to install and the installer doesn't responds after

**Comment by joinalahmed** [comment]
@Ghostkeeper  i've tried out the patches on #63

**Comment by thopiekar** [comment]
Are you using the software center?

**Comment by joinalahmed** [comment]
@thopiekar  i'm using software install to install

**Comment by joinalahmed** [comment]
![screenshot from 2016-07-01 18-36-43](https://cloud.githubusercontent.com/assets/14151006/16522325/cfd1307e-3fba-11e6-8d9f-dd32d3bc3569.png)
when i click on install, it shows installing for a second and again shows install button, and a widget appears on task bar that shows waiting to install for forever

**Comment by thopiekar** [comment]
ah, ok, Canonical just changed the name. It was called "Software center" before. This is a known problem with that application.

**Comment by joinalahmed** [comment]
any possible solutions for the problem

**Comment by thopiekar** [comment]
Use "GDebi" for the installation

**Comment by joinalahmed** [comment]
it shows the package is installed, but where?

**Comment by thopiekar** [comment]
You should find the Cura icon in the menu: Graphics > Cura

**Comment by joinalahmed** [comment]
not finding it

**Comment by thopiekar** [comment]
go to terminal and execute: cura
It it also says there is no "cura" then GDebi was lying that it was installed.

**Comment by joinalahmed** [comment]
![screenshot from 2016-07-01 19-03-45](https://cloud.githubusercontent.com/assets/14151006/16522981/8ec4b0de-3fbe-11e6-809d-823bdb454e4d.png)

**Comment by kenjo** [comment]
use the command line instead so we can see what actually happens. 
to see if cura is installed. 

dpkg -l cura

to install the deb file 

dpkg -i cura.xxxx.deb

**Comment by thopiekar** [comment]
@joinalahmed, @kenjo: You sure you are interested in testing this: https://github.com/Ultimaker/cura-build/pull/68 :wink:

**Comment by ryneches** [comment]
Can confirm. Cura's package specification is completely wrong; it checks for python version >=3.4, which is satisfied by python3.5 which allows it to be installed on Ubuntu 16.04. However, Cura then invokes python3.4 explicitly, which has been depreciated and does not exist.

There is so much broken here I don't even know where to stat. I'm giving up and sticking with my ancient pre-1.0 build.

**Comment by Elemino** [comment]
Hello, 

I am running ubuntu 16.04 LTS 64 bit.  Installed Cura and the icon appears on my lunch bar. When I click it, nothing happens. I am not sure where to start troubleshooting. I wonder if you have any advice. 

Thanks a lot!

**Comment by Ghostkeeper** [comment]
Did it produce a file at `~/.local/share/cura/cura.log`? If so, you could post it here.

Otherwise, you could try starting Cura from a command line. Simply type `cura` in a command line and it should start.

**Comment by Elemino** [comment]
Sorry for the late response. No. It didn't. When I try to run 'cura' I get the following:

/usr/bin/cura: line 3: python3.4: command not found

I did try 'apt-get upgrade cura'. Thanks!

**Comment by Ghostkeeper** [comment]
Ah, that clarifies things. It appears to be the same problem as joinalahmed had in this thread.

We fixed this in a different way for Cura 2.4. Instead of releasing .deb packages, we will distribute AppImage files. These should contain all dependencies, including Python. We also upgraded to Python 3.5 in the meanwhile, by the way, so even if we didn't make that change, we wouldn't get the same error.

So... fixed for version 2.4!
